### ISSUE TITLE
Favorite / recent searches in search window

### ISSUE THREAD

**Issue by Viiprogrammer** [ankitskvmdam/clean-jsdoc-theme#137]
**Is your feature request related to a problem? Please describe.**

No

**Describe the solution you'd like**

Add favorite searches / recent like [here](https://vuejs.org/):

![изображение](https://user-images.githubusercontent.com/17622604/183291783-4f0bc10f-e64b-4498-afad-c52747a18062.png)

**Additional context**

Their store this data in [Local Storage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)

You may need to add a check availability for the current link in the documentation when opening a window

**Comment by ankitskvmdam** [comment]
Yes, this is something nice to have. I will try to find some time this week to work on this feature. Or would you like to work on this?

**Comment by Viiprogrammer** [comment]
> Yes, this is something nice to have. I will try to find some time this week to work on this feature. Or would you like to work on this? 

I still have a lot of work to do, but I'll try, maybe closer to the weekend, I'll write here if anything 🙃
